CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.4.0 (2019-03-19)
+
+- Added `SkyAutocompleteSearchFunctionFilter`, `SkySectionedFormComponent`, `SkyEmailValidationDirective`, and `SkyNavbarModule` types to package map. [#35](https://github.com/blackbaud/skyux-sdk-builder-plugin-migrate/pull/26)
```

Comment:
The link here is wrong. We start with the correct PR number, but the URL points to the wrong PR. It should point to https://github.com/blackbaud/skyux-sdk-builder-plugin-migrate/pull/35. ... I'll update as soon as I check the rest of the changelog entry.